openforcefields/offxml/opc-1.0.0.offxml

```diff
@@ -134,6 +134,6 @@
         <Constraint smirks="[#1:1]-[#8X2H2+0]-[#1:2]" id="c-opc-H-O-H" distance="1.3712051 * angstrom ** 1"></Constraint>
     </Constraints>
     <VirtualSites version="0.3" exclusion_policy="parents">
-        <VirtualSite smirks="[#1:2]-[#8X2H2+0:1]-[#1:3]" epsilon="0.0 * kilocalories_per_mole ** 1" type="DivalentLonePair" match="once" distance="-0.15939833 * angstrom ** 1" outOfPlaneAngle="0.0 * degree ** 1" inPlaneAngle="None" charge_increment1="0.0 * elementary_charge ** 1" charge_increment2="0.679142 * elementary_charge ** 1" charge_increment3="0.679142 * elementary_charge ** 1" rmin_half="1.0 * angstrom ** 1" name="EP"></VirtualSite>
+        <VirtualSite smirks="[#1:2]-[#8X2H2+0:1]-[#1:3]" epsilon="0.0 * kilocalorie_per_mole ** 1" type="DivalentLonePair" match="once" distance="-0.15939833 * angstrom ** 1" outOfPlaneAngle="0.0 * degree ** 1" inPlaneAngle="None" charge_increment1="0.0 * elementary_charge ** 1" charge_increment2="0.679142 * elementary_charge ** 1" charge_increment3="0.679142 * elementary_charge ** 1" rmin_half="1.0 * angstrom ** 1" name="EP"></VirtualSite>
```

Comment:
(blocking) It's super important to me that we never change the contents of a versioned FF after release. I know this change is just cosmetic but I don't want to let any precedent through.

Reply:
Reverted changes to opc-1.0.0 and made opc-1.0.1 in [3558ecd](https://github.com/openforcefield/openff-forcefields/pull/69/commits/3558ecd1186a272c145234841b519a214ecb17aa)

Reply:
Thanks for reviewing and merging, Jeff! This rationale makes sense to me, and I'll keep this in mind for future PRs.